pandas/tests/util/test_deprecate_nonkeyword_arguments.py

```diff
@@ -155,3 +165,17 @@ def test_class():
     )
     with tm.assert_produces_warning(FutureWarning, match=msg):
         Foo().baz("qux", "quox")
+
+
+def test_deprecated_keywords():
```

Comment:
This file is for testing the decorator itself, these tests should go in `pandas/tests/groupby/test_function`.